spice.js

```diff
@@ -1,1944 +0,0 @@
-		* @desc Translate the name of a body or object to the corresponding SPICE integer ID code.
-		* @param {name} name - The common name of the body.
-		* @returns {number | undefined} The SPICE ID of the body if it exists, otherwise undefined.
-		*/
-		bodn2c: function(name){
-			var code_ptr = this.Module._malloc(integer_size);
-			var found_ptr = this.Module._malloc(integer_size);
-			this.Module.ccall(
-				'bodn2c_c',
-				null,
-				["string","number","number",],
-				[name,code_ptr,found_ptr,]
-			);
-			var found = this.Module.getValue(found_ptr,integer_type);
-			var code = this.Module.getValue(code_ptr,integer_type);
-			this.Module._free(found_ptr);
-			this.Module._free(code_ptr);
-			if(found){	//If code exists
-				return code; //Return it
-			} else {
-				return undefined;
-			}
-		},
-
-		/*bods2c:    Translate a string containing a body name or ID code to an integer
-		   code.
-		*/
-		/** @memberof SPICE
-		* @func bods2c
-		* @desc Translate the name of a body or object to the corresponding SPICE integer ID code.
-		* @param {name} name - The common name of a body or its code as a string.
-		* @returns {number | undefined} If a body name was passed in, the SPICE ID of the body if it exists, otherwise undefined. If a string number was passed in, the number as an integer.
-		*/
-		bods2c: function(name){
-			var code_ptr = this.Module._malloc(integer_size);
-			var found_ptr = this.Module._malloc(integer_size);
-			this.Module.ccall(
-				'bods2c_c',
-				null,
-				["string","number","number",],
-				[name,code_ptr,found_ptr,]
-			);
-			var found = this.Module.getValue(found_ptr,integer_type);
-			var code = this.Module.getValue(code_ptr,integer_type); 
-			this.Module._free(code_ptr);
-			this.Module._free(found_ptr);
-
-			if(found){	//If the code exists
-				return code;//Return it
-			} else {
-				return undefined;
-			} 
-
-		},
-
-		/*bodvcd:  
-		   Fetch from the kernel pool the double precision values of an item
-		   associated with a body, where the body is specified by an integer ID
-		   code.
-		 */
-		/** @memberof SPICE
-		* @todo Document and test this! 
-		*/
-		bodvcd: function(bodyid,item,maxn){
-			var dim_ptr = this.Module._malloc(8);
-			var values_ptr = this.Module._malloc(8*maxn);
-			this.Module.ccall(
-				'bodvcd_c',
-				null,
-				["number","string","number","number","number",],
-				[bodyid,item,maxn,dim_ptr,values_ptr,]
-			);
-			var ret = [];
-			for (var i = 0; i < this.Module.getValue(dim_ptr,integer_type); i++){
-				ret.push(this.Module.getValue(values_ptr+i*8, 'double'));
-			}
-			this.Module._free(dim_ptr);
-			this.Module._free(values_ptr);
-
-			return ret;
-		},
-
-		/*bodvrd:  
-		   Fetch from the kernel pool the double precision values  
-		   of an item associated with a body. 
-		 */
-		/** @memberof SPICE
-		* @todo Document and test this! 
-		*/
-		bodvrd: function(body, item, maxn) {
-			var valuesptr = this.Module._malloc(8*maxn);
-			var dimptr = this.Module._malloc(2);
```

Comment:
I don't know why this was set to "2" previously (changed to INT_SIZE = 4 now) -- the [docs state dim is an INT](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/bodvrd.html).